specification/redisenterprise/resource-manager/readme.az.md

```diff
@@ -0,0 +1,24 @@
+## AZ
+
+These settings apply only when `--az` is specified on the command line.
+
+``` yaml $(az) && $(target-mode) != core
+az:
+    extensions: redisenterprise
+    namespace: azure.mgmt.redisenterprise
+    package-name: azure-mgmt-redisenterprise
+az-output-folder: $(azure-cli-extension-folder)/src/redisenterprise
+python-sdk-output-folder: "$(az-output-folder)/azext_redisenterprise/vendored_sdks/redisenterprise"
+# add additinal configuration here specific for Azure CLI
+# refer to the faq.md for more details
+```
+``` yaml $(az) && $(target-mode) == core
+az:
+    extensions: redisenterprise
+    namespace: azure.mgmt.redisenterprise
+    package-name: azure-mgmt-redisenterprise
+az-output-folder: $(azure-cli-folder)/src/azure-cli/azure/cli/command_modules/redisenterprise
+python-sdk-output-folder: "$(az-output-folder)/vendored_sdks/redisenterprise"
+# add additinal configuration here specific for Azure CLI
```

Comment:
is that a copy/paste typo? if so maybe not worth fixing but...

Reply:
I copied from here as instructed: https://github.com/Azure/autorest.az/blob/master/doc/how-to-author-readme-file.md

But yes I see the typo now "additinal"